aws-lc-rs/src/kem.rs

```diff
@@ -256,14 +257,8 @@ where
     }
 }
 
-mod types {
-    pub struct EncapsulationKeyBytesType {
-        _priv: (),
-    }
-}
-
-/// KEM Encapsulation Key Bytes.
-pub type EncapsulationKeyBytes = Buffer<'static, types::EncapsulationKeyBytesType>;
+use paste::paste;
+generated_encodings!(EncapsulationKeyBytes);
```

Comment:
The `generated_encodings` macro declares the type it creates as public.  Since this type is specific to the `kem` module, we only wanted it exported from here.

Reply:
Ahh overlooked that detail, thanks!